samples/drivers/i2s/i2s_codec/src/main.c

```diff
@@ -193,7 +193,6 @@ int main(void)
 #else
 				/* If not using DMIC, play a sine wave 440Hz */
 				mem_block = (void *)&__16kHz16bit_stereo_sine_pcm;
-				block_size = __16kHz16bit_stereo_sine_pcm_len;
```

Comment:
Perhaps a build assertion ensuring that `block_size <= __16kHz16bit_stereo_sine_pcm_len` would be helpful here.